code/__DEFINES/lunar_defines.dm

```diff
@@ -0,0 +1,9 @@
+//xenowear for the loadout tab
```

Comment:
These defines shouldn't be lumped together under a signle lunar_defines.dm, I'd rather them be added to bottom of their proper respective files with a lunaredit tag to keep the defines organized.